ERCS/erc-7876.md

```diff
@@ -0,0 +1,591 @@
+        "Registry": {
+          "address": "0x57928ff7b0BBc3Ee4D84481e320DdB8B941f986A",
+          "blockCreated": 1234567,
+          "abiUrl": "./Registry.sol/Registry.json"
+        },
+        "OwnerWallet": {
+          "address": "0xC12237E57B088e9191BD8054Df4f5B772646a4B6",
+          "blockCreated": 1
+        }
+      }
+    },
+    "11155111": {
+      "name": "Sepolia",
+      "testnet": true,
+      "nativeCurrency": {
+        "name": "Sepolia Ether",
+        "symbol": "ETH",
+        "decimals": 18
+      },
+      "rpcs": {
+        "main": {
+          "url": "https://sepolia-rpc.example.com"
+        },
+        "backup": {
+          "url": "https://sepolia-rpc.backup.example.com"
+        }
+      },
+      "relations": {
+        "mainnetChainId": 1,
+        "parentChainId": null
+      },
+      "explorers": {
+        "megascan": {
+          "root": "https://sepolia.example.org",
+          "block": "/block/:number",
+          "address": "/address/:address",
+          "tx": "/tx/:tx",
+          "nft": "/nft/:address/:token"
+        },
+        "marketplace": {
+          "root": "https://testnets.nft.marketplace/networks/sepolia",
+          "block": null,
+          "address": "/:address",
+          "tx": null,
+          "nft": "/:address/:token"
+        }
+      },
+      "contracts": {
+        "Registry": {
+          "address": "0xE13471e6E5d11205AF290261f42108f89dCae72E",
+          "blockCreated": 183882,
+          "abiUrl": "./Registry.sol/Registry.json" 
+        },
+        "OwnerWallet": {
+          "address": "0xC12237E57B088e9191BD8054Df4f5B772646a4B6",
+          "blockCreated": 1
+        }
+      }
+    }
+  }
+}
+```
+
+*Note*: It is **RECOMMENDED** to use a two-space indentation format for better readability and debugging.
+
+### Configuration Loader in Swift
+
+Here is an example implementation of a configuration loader in Swift Programming Language, demonstrating how to load and parse a configuration file compliant with this standard:
+
+``` swift
+import Foundation
+
+struct BlockchainConfig: Codable, LoggerProvider {
+    struct NetworkConfig: Codable {
+        struct Currency: Codable {
+            let name: String
+            let symbol: String
+            let decimals: Int
+        }
+
+        struct RPC: Codable {
+            let url: String
+        }
+
+        struct Relations: Codable {
+            let mainnetChainId: Int?
+            let parentChainId: Int?
+        }
+
+        struct Explorer: Codable {
+            let root: String
+            let block: String?
+            let address: String?
+            let tx: String?
+            let nft: String?
+        }
+
+        struct Contract: Codable {
+            let address: String
+            let blockCreated: Int
+        }
+
+        let name: String
+        let testnet: Bool
+        let nativeCurrency: Currency
+        let rpcs: [String: RPC]
+        let relations: Relations
+        let verify: String
```

Comment:
The NetworkConfig struct includes a verify property that isn't defined in the standard. Clarify if this is intentional

Reply:
That was an extension in my own project.

```suggestion
```